windows/dockerfile-to-powershell-generated-sample.ps1

```diff
@@ -185,8 +185,8 @@ function Update-SessionEnvironment {
 # Content: ENV EMBEDDED_PIP_VERSION "20.3.4"
 [System.Environment]::SetEnvironmentVariable('EMBEDDED_PIP_VERSION','20.3.4')
 
-# Content: ENV CACERTS_HASH "08df40e8f528ed283b0e480ba4bcdbfdd2fdcf695a7ada1668243072d80f8b6f"
-[System.Environment]::SetEnvironmentVariable('CACERTS_HASH','08df40e8f528ed283b0e480ba4bcdbfdd2fdcf695a7ada1668243072d80f8b6f')
+# Content: ENV CACERTS_HASH "6ed95025fba2aef0ce7b647607225745624497f876d74ef6ec22b26e73e9de77"
+[System.Environment]::SetEnvironmentVariable('CACERTS_HASH','6ed95025fba2aef0ce7b647607225745624497f876d74ef6ec22b26e73e9de77')
 
 # Content: ENV CACERTS_VERSION "2022-04-26"
 [System.Environment]::SetEnvironmentVariable('CACERTS_VERSION','2022-04-26')

```

Comment:
These also need an update

Reply:
Right ! Fixed in [4ea6287](https://github.com/DataDog/datadog-agent-buildimages/pull/261/commits/4ea62870d217bf55226031544fb2422fd53283b3), thanks ! 🤠